recipe/meta.yaml

```diff
@@ -76,6 +76,7 @@ requirements:
     - qiskit >=0.44.0
     - scipy >=1.0
     - cuda-version >={{ cuda_major }}.2,<{{ cuda_major+1 }}  # [(cuda_compiler_version or "").startswith("11")]
+    - cuda-version >={{ cuda_major }}.0,<{{ cuda_major+1 }}  # [(cuda_compiler_version or "").startswith("12")]
     - cudatoolkit                                            # [(cuda_compiler_version or "").startswith("11")]
```

Comment:
@wshanks I was looking at the Aer codebase. I see that Aer doesn't direct call any of the CUDA libraries (cublas, cuSolver, cusparse, ...), so why did you add dependencies on them for wheel users? I feel life could be very simple here :D

Reply:
Ah, ok I probably know why, cuquantum wheels do not declare dependencies on the cuda wheels, so it becomes the job of our downstream. We'll figure it out.

Reply:
(cc: @yangcal we should discuss this)

Reply:
@leofang Just to clarify my role since you used "you" there -- I work on qiskit-experiments which uses Qiskit Aer to run its tests but I am not on the Aer team. Since Qiskit has decided to support only PyPI as an official distribution mechanism, I treat my work on conda-forge as something on do my own time, because I prefer to use conda, not an official Qiskit effort.

Reply:
@wshanks my apology for poor choice for my wording on an exhausting Friday. By "you" I had in mind the Aer team, and I wasn't aware you're not part of them. Thank you for clarifying and sorry again for the misunderstanding.